grug/testing/tests/cron.rs

```diff
@@ -143,26 +143,34 @@ fn cronjob_works() {
     // Make some blocks.
     // After each block, check that Jake has the correct balances.
     for balances in [
-        // Block time: 6
+        // Block time: 5
         //
         // cron1 sends 1 uatom, rescheduled to 6
         Balances {
             uatom: 1,
             uosmo: 0,
             umars: 0,
         },
-        // Block time: 7
+        // Block time: 6
         //
         // cron1 sends 1 uatom, rescheduled to 7
-        // cron2 sends 1 uosmo, rescheduled to 9
         Balances {
             uatom: 2,
+            uosmo: 0,
+            umars: 0,
+        },
+        // Block time: 7
+        //
+        // cron1 sends 1 uatom, rescheduled to 8 (it runs out of coins here)
+        // cron2 sends 1 uosmo, rescheduled to 9
+        Balances {
+            uatom: 3,
```

Comment:
now we move cronjob to end of block, the behavior of this test case changes & needs to be updated